Makefile

```diff
@@ -1,8 +1,8 @@
 -include .env
 export
 
-START_BLOCK ?= 4051031
-STOP_BLOCK ?= 0
+START_BLOCK ?= 19028181
+STOP_BLOCK ?= 19028182
```

Comment:
Are these just for development?

Reply:
yes it's just for verifying the data is properly indexed in the example blocks